### ISSUE TITLE
Loss increases after some iterations

### ISSUE THREAD

**Issue by oReano** [rosinality/glow-pytorch#1]
Hi! I tried to run your code. The network starts out training well and decreases the loss but after a few iterations, the loss just starts to increase.  I tried smaller initial learning rate, but still not working. Could you please help me fix this problem? Thank you!

**Comment by rosinality** [comment]
As I haven't experimented extensively, I can't suggest direct solutions...I recommend to increase batch sizes, and use different n_bits parameters.

You can use DataParallel to increase batch sizes. But as ActNorm use batch statistics to initialize parameters, you should do 1 forward step with full batch sizes in 1 gpus.

**Comment by oReano** [comment]
I tried to initialize ActNorm before using DataParallel, and it works. Thank you for your valuable suggestion :)
